server/play.js

```diff
@@ -1,5 +1,10 @@
-const knex = require("./utils/dbConfig");
+const db = require("./utils/dbConfig");
 
-knex("users")
-  .select("*")
-  .then((result) => console.log(result));
+// knex("users")
+//   .select("*")
+//   .then((result) => console.log(result));
+  db("products").insert({
+    name: "x",
+    description: "dcf",
+    category_id: 2,
+}).returning("products.id").then((result)=> console.log(result));
```

Comment:
why did you add these play.js file changes here?
this is just for your usage right?